### PR TITLE
fixes import plugins for browser

### PR DESCRIPTION
**Pull request by ayush2k**
Fixes import of plugin using browserify;
fixes unintentional changes from plugabilly

`DCO 1.1 Signed-off-by: {{Ayush Gupta}} <{{ayush2k@gmail.com}}>`

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -49,6 +53,47 @@ exports.splitPop = (input, splitter) => {
...
+    dir = [dir];
+  }
+
+  // Adds path of node_modules to directory
+  if (!Array.isArray(dir)) {
+    dir = [root];
+  }
+
+  // Checks if path to node_modules exists
+  else if (dir.filter(value => {
```
Comment (scottnath) [inline lib/util.js]: would this work?
`else if (!dir[root])`

File: lib/util.js [hunk 2]
```diff
@@ -49,6 +53,47 @@ exports.splitPop = (input, splitter) => {
...
+  let dir = _.get(globalConfig, 'content.plugins.directory', []);
+
+  // Check if configuration is set to string
+  if (typeof dir === 'string') {
+    dir = [dir];
+  }
+
+  // Adds path of node_modules to directory
+  if (!Array.isArray(dir)) {
+    dir = [root];
```
Comment (scottnath) [inline lib/util.js]: this is not covered in your tests. Please add a test for this.

File: lib/util.js (hunk 1 above)
Comment (ayush2k) [inline lib/util.js]: No, dir is an array.
Cannot use string to get element

File: lib/util.js (hunk 2 above)
Comment (ayush2k) [inline lib/util.js]: added new test with directory as boolean. It covers this line of code.

File: lib/util.js (hunk 1 above)
Comment (scottnath) [inline lib/util.js]: actually...`dir.indexOf`

**Comment by scottnath** [comment]
@ayush2k can you please change the name of the util.js file to utils.js? (we don't want confusion with the node `util` module itself.)

File: lib/util.js (hunk 1 above)
Comment (ayush2k) [inline lib/util.js]: Done. 
I think it would be more efficient.

**Comment by ayush2k** [comment]
@scottnath changes util to utils

**Pull request merged**